vars/getCheckoutInfo.groovy

```diff
@@ -0,0 +1,33 @@
+def call() {
+
+  def repoInfo = [
+    branch: env.ghprbSourceBranch ?: params.branch_specifier ?: 'missing branch_specifier',
```

Comment:
I'm trying to think of a good fallback here, because this is almost guaranteed to be a gotcha at some point... If a non-PR job isn't using a  `branch_specifier` build parameter, this part will fail, and `GIT_BRANCH` on the outside will be `missing branch_specifier`. Could it at least be null so that it could be checked on the outside, rather than pass that string all the way through the pipeline?